docs/upgrade/10_to_11/migrate-indices.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+SERVER="http://localhost:9200"
+USER=""
+PASSWORD=""
+
+NUMBER_OF_REPLICAS=0  # change this if you use a cluster
+AUTO_EXPAND_REPLICAS="false"
+
+TIMEOUT="60s"
+
+OLD_INDEX="adminui"  # Do not use external API index as source as that one doesn't contain themes!
+NEW_INDEX="opencast"
+
+# sub indices
+VERSION="version"
+EVENT="event"
+SERIES="series"
+THEME="theme"
+
+set -e
```

Comment:
```suggestion
set -eu
```

Plus all the `$USER:$PASSWORD` should be in quotes